portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/Exporter.java

```diff
@@ -22,6 +22,9 @@
 /** Exports data from a source service. */
 public interface Exporter<A extends AuthData, T extends DataModel> {
 
+  /** The key associated with this service. */
+  String getServiceId();
```

Comment:
Per off line conversation, let's revert this

Reply:
Done